pkg/mounter/mounter-rclone.go

```diff
@@ -49,6 +49,14 @@ const (
 	envAuth        = "true"
 )
 
+var (
+	MakeDir    = os.MkdirAll
```

Comment:
I think we should have kept in `mounter.go` since its being used in both files `mounter-s3fs.go` and `mounter-rclone.go`